managed/src/main/java/com/yugabyte/yw/controllers/InstanceTypeController.java

```diff
@@ -57,6 +57,17 @@ public Result list(UUID customerUUID, UUID providerUUID) {
 
     try {
       instanceTypeList = InstanceType.findByProvider(provider);
+      // setting up the volumeCount to 1 for instanceType starting with c5/c4
+      for (InstanceType instanceType: instanceTypeList) {
+        if (provider.code.equals(aws.toString()) &&
+          ((instanceType.idKey.instanceTypeCode.startsWith("c5")) ||
```

Comment:
This is incorrect. `c5d` instances are similar to i3 instances in that they use NVMe drives. So we want to use the value present in the details.

Reply:
Thanks @Arnav15  for confirming this. Will do this only for c4/C5 instance type  which won't include c5d.
Will do necessary changes and update CR.